library/mono

```diff
@@ -1,21 +1,30 @@
-# maintainer: Jo Shields <jo.shields@xamarin.com> (@directhex)
-# maintainer: Alexander Köplinger <alkpli@microsoft.com> (@akoeplinger)
+# this file is generated via https://github.com/mono/docker/blob/master/generate-stackbrew-library.sh
 
-3.12.1: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 3.12.1
-3.12.0: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 3.12.1
-3.12: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 3.12.1
-3: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 3.12.1
+Maintainers: Jo Shields <jo.shields@xamarin.com> (@directhex),
+             Alexander Köplinger <alkpli@microsoft.com> (@akoeplinger)
+GitRepo: https://github.com/mono/docker.git
 
-4.8.0.524: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 4.8.0.524
-4.8.0: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 4.8.0.524
-4.8: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 4.8.0.524
-4: git://github.com/mono/docker@0d987d93235630e05a14983c7f87500ab33c90da 4.8.0.524
+Tags: 5.2.0.215, 5.2.0, 5.2, 5, latest
+Architectures: amd64, i386, arm32v7, arm64v8
+GitCommit: c895836e0e8cb7ee0d8124f360d2febcf376019d
+Directory: 5.2.0.215
 
-5.0.0.100: git://github.com/mono/docker@d7a8871afc770695b81977272792d8966340939e 5.0.0.100
-5.0.0: git://github.com/mono/docker@d7a8871afc770695b81977272792d8966340939e 5.0.0.100
+Tags: 5.2.0.215-slim, 5.2.0-slim, 5.2-slim, 5-slim, latest-slim
```

Comment:
From https://github.com/docker-library/official-images#tags-and-aliases:

> In Docker, the `latest` tag is a special case, but it's a bit of a misnomer; `latest` really is the "default" tag. When one does `docker run xyz`, Docker interprets that to mean `docker run xyz:latest`. Given that background, no other tag ever contains the string `latest`, since it's not something users are expected or encouraged to actually type out (ie, `xyz:latest` should really be used as simply `xyz`). Put another way, having an alias for the "highest 2.2-series release of XYZ" should be `xyz:2.2`, not `xyz:2.2-latest`. Similarly, if there is an Alpine variant of `xyz:latest`, it should be aliased as `xyz:alpine`, not `xyz:alpine-latest` or `xyz:latest-alpine`.

So this last `latest-slim` tag should simply be `slim` instead. :+1: